sros2/sros2/api/__init__.py

```diff
@@ -189,12 +214,21 @@ def create_ecdsa_param_file(path):
     run_shell_command('%s ecparam -name prime256v1 > %s' % (openssl_executable, path))
```

Comment:
Can we get rid of this function, now? Or do we use it/that file elsewhere?

Reply:
We are still using `create_ecdsa_param_file` in one other place, my thinking is the next pass on this shell-removal process would remove this function and file. (the linter would complain if it was unused)